internal/uri/http.go

```diff
@@ -23,6 +23,10 @@ func (r *HTTPURIReader) Read(uri string) (io.ReadCloser, error) {
 	}
 	request.Header.Add("Accept-Encoding", "gzip")
 
```

Comment:
I wonder if it would be better if this was set via http transport as it's done with TLS config - [see here](https://github.com/prymitive/karma/blob/master/internal/alertmanager/upstream.go#L115)
Googling around seems to suggest it could be a better path indeed - https://stackoverflow.com/a/47466042.
But I can look into refactoring it later